src/components/Article.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => (
```

Comment:
Check this
https://mateacademy.slack.com/archives/C02CSGP1KD1/p1636008283104500?thread_ts=1636006111.100400&cid=C02CSGP1KD1